ci/travis_integration_run.sh

```diff
@@ -12,5 +12,5 @@ cd qa/integration
 # due to some sideeffects of the seccomp policy interfering with
 # the docker daemon
 # See prepare_offline_pack_spec.rb for details
-rspec --tag ~offline
-rspec --tag offline
+bundle exec rspec --tag ~offline
+bundle exec rspec --tag offline
```

Comment:
is `bundle exec` required, or just more correct ?

Reply:
@jakelandis required, without it the jar dependencies (from the repo itself instead of some LS 5.x) aren't loaded properly.